zookeeper-docs/src/main/resources/markdown/zookeeperAdmin.md

```diff
@@ -1531,7 +1531,8 @@ and [SASL authentication for ZooKeeper](https://cwiki.apache.org/confluence/disp
 * *ssl.keyStore.type* and *ssl.quorum.keyStore.type* :
     (Java system properties: **zookeeper.ssl.keyStore.type** and **zookeeper.ssl.quorum.keyStore.type**)
     **New in 3.5.5:**
-    Specifies the file format of client and quorum keystores. Values: JKS, PEM, PKCS12 or null (detect by filename).
+    Specifies the file format of client and quorum keystores. Values: JKS, PEM, PKCS12, BCFKS or null (detect by filename).
+    The format BCFKS was added in ZooKeeper 3.6.3.
```

Comment:
Nit/question: For readability add **New in 3.6.3** Support for BCFKS?

Reply:
I meant to say add it the same way the whole feature is introduced, with a separate double asterix header, but theres already a discussion on this.

Reply:
yes, maybe this would be a more consistent way... I updated this, please check the new version